docs/guides/references/experiments.mdx

```diff
@@ -25,13 +25,74 @@ configuration to Cypress.
 | `experimentalInteractiveRunEvents`            | `false` | Allows listening to the [`before:run`](/api/plugins/before-run-api), [`after:run`](/api/plugins/after-run-api), [`before:spec`](/api/plugins/before-spec-api), and [`after:spec`](/api/plugins/after-spec-api) events in the [setupNodeEvents](/guides/tooling/plugins-guide#Using-a-plugin) function during interactive mode. |
 | `experimentalMemoryManagement`                | `false` | Enables support for improved memory management within Chromium-based browsers.                                                                                                                                                                                                                                                 |
 | `experimentalModifyObstructiveThirdPartyCode` | `false` | Whether Cypress will search for and replace obstructive code in third party `.js` or `.html` files. NOTE: Setting this flag removes [Subresource Integrity (SRI)](https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity).                                                                                |
 | `experimentalSourceRewriting`                 | `false` | Enables AST-based JS/HTML rewriting. This may fix issues caused by the existing regex-based JS/HTML replacement algorithm. See [#5273](https://github.com/cypress-io/cypress/issues/5273) for details.                                                                                                                         |
 | `experimentalWebKitSupport`                   | `false` | Enable experimental support for running tests in WebKit. When set, installs of `playwright-webkit` will be detected and available in Cypress. See [Launching Browsers](/guides/guides/launching-browsers#WebKit-Experimental) for more information.                                                                            |
 
+### Experimental CSP Allow List
+
+Cypress will (by default) strip all CSP headers from the response before it is
+sent to the browser. This option allows for more granular control over which CSP
+directives are stripped from the CSP response headers, allowing you to test your
+application with CSP enabled.
+
+#### Strip All CSP Headers
+
+The default value of `experimentalCspAllowList` is `false`. This will remove all
+CSP headers from the response before it is sent to the browser. This option
+should be used if you do not depend on CSP for any tests in your application.
+
+#### Strip Minimum CSP Directives
+
+If you need to test your application with CSP enabled, you can set the
+`experimentalCspAllowList` option to `true`. This will allow all CSP headers to
+be sent to the browser _*except*_ those that could prevent Cypress from
+functioning normally. The following CSP directives are automatically stripped
```

Comment:
I think it would be clear to break the stripped directive table out into the list that is always stripped and then the list that can be configured:

```suggestion
functioning normally. The following CSP directives will always be stripped:

| Stripped Directive           |  Reason                                                           |
| ---------------------------- | ---------------------------------------------------------------- |
| `frame-ancestors`        |    Prevents Cypress from loading a test application into an iframe. |
| `require-trusted-types-for` | Might prevent Cypress from rewriting the DOM.                    |
| `trusted-types`           | Could cause Cypress injections to be marked as untrusted.        |

When `experimentalCspAllowList=true` the following will also be stripped unless specified.

| Stripped Directive           | Allowable | Reason                                                           |
| ---------------------------- | --------- | ---------------------------------------------------------------- |
| `default-src`                | Yes       | Conditionally prevents Cypress from loading scripts and running. |
| `sandbox`                    | Yes       | Can restrict access to script and iframe functionality.          |
| `form-action`                | Yes       | Can prevent Cypress from monitoring form events.                 |
| `navigate-to`                | Yes       | Affects Cypress' ability to navigate to different URLs.          |
| `script-src`                 | Yes       | Conditionally prevents Cypress from loading scripts and running. |
| `script-src-elem`            | Yes       | Conditionally prevents Cypress from loading scripts and running. |
```

Reply:
@emilyrohrbough I've made some modificaitons to the overall structure to include this change. Mainly being iterative with an initial table that demonstrates the configuration options and linking to the appropriate sections.